.github/workflows/5_e2e_tests.yml

```diff
@@ -0,0 +1,46 @@
+name: Integration tests 
```

Comment:
Only VSCode extension or also LSP?

Reply:
This runs the e2e tests the vscode folder with the published LSP nom package

Reply:
Might be worth renaming to make this crystal clear.